src/views/partials/breaches.js

```diff
@@ -31,11 +31,17 @@ function createBreachRows (data) {
         addedDate: longDate.format(addedDate),
         dataClasses: longList.format(dataClassesTranslated)
       })
+      const logoPath = logos.get(breach.Domain) ?? '/images/fallback-logo.svg'
 
       return `
       <details class='breach-row' data-status=${status} data-email=${account.email} ${isHidden ? 'hidden' : ''}>
         <summary>
-          <span>${breach.Title}</span>
+          <span>
+            <a href='https://${breach.Domain}' target='_blank' class='breach-company'>
+              <img src='${logoPath}' alt='' class='breach-logo' height='32' />
+              ${breach.Title}
+            </a>
+          </span>
```

Comment:
It looks like wrapping the breach title (company name) in `<a>` broke the ellipses.  As mentioned, I think we should remove the link altogether – hopefully that resolves the issue.

before:
<img width="205" alt="Screen Shot 2023-02-23 at 3 06 52 PM" src="https://user-images.githubusercontent.com/14863500/221051522-599858b1-e62f-49be-a47d-e55c0d513765.png">
 
after:
<img width="218" alt="Screen Shot 2023-02-23 at 2 58 44 PM" src="https://user-images.githubusercontent.com/14863500/221051527-a157ad9d-7ad8-4d4d-aa0e-6c0a891f0449.png">

Reply:
Ah yes, wasn't aware that we did that, but removing the links should resolve that. If we should keep the links, I'll make sure to transplant the ellipsis.